types/index.d.ts

```diff
@@ -2805,6 +2805,19 @@ export interface MathJsInstance extends MathJsFactory {
    */
   prod(A: MathCollection): MathScalarType
 
+  /**
+   * @param A An array of scalar values
+   * @param probOrN prob is the order of the quantile, while N is the
+   * amount of evenly distributed steps of probabilities; only one of
+   * these options can be provided
+   * @param sorted =false is data sorted in ascending order
+   * @returns Quantile(s)
+   */
+  quantileSeq<T extends MathScalarType>(
```

Comment:
Thanks, this looks good! Can you add a few cases to [/test/typescript-tests/testTypes.ts](https://github.com/josdejong/mathjs/blob/develop/test/typescript-tests/testTypes.ts) to ensure the new typings actually work?

Reply:
Sure, have done ✅

Ready for re-review

Reply:
Thanks 👌